extensions/amp-story/1.0/pagination-buttons.js

```diff
@@ -222,50 +208,40 @@ export class PaginationButtons {
   onCurrentPageIndexUpdate_(pageIndex) {
     const totalPages = this.storeService_.get(StateProperty.PAGE_IDS).length;
 
-    if (pageIndex === 0) {
-      this.backButton_.updateState(BackButtonStates.HIDDEN);
-    }
-
-    if (pageIndex > 0) {
-      this.backButton_.updateState(BackButtonStates.PREVIOUS_PAGE);
-    }
+    // Hide back button if no previous page.
+    this.setEnabled_(this.backButton_, pageIndex > 0);
 
     if (pageIndex < totalPages - 1) {
-      this.forwardButton_.updateState(ForwardButtonStates.NEXT_PAGE);
-    }
-
-    if (pageIndex === totalPages - 1) {
+      this.forwardButton_.updateState(ButtonStates.NEXT_PAGE);
+    } else {
       const viewer = Services.viewerForDoc(this.ampStory_.element);
       if (viewer.hasCapability('swipe')) {
-        this.forwardButton_.updateState(ForwardButtonStates.NEXT_STORY);
+        this.forwardButton_.updateState(ButtonStates.NEXT_STORY);
       } else {
-        this.forwardButton_.updateState(ForwardButtonStates.REPLAY);
+        this.forwardButton_.updateState(ButtonStates.REPLAY);
       }
     }
   }
 
+  /**
+   * Updates element based on visibility state.
+   * @param {!PaginationButton} paginationButton
+   * @param {boolean} isVisible
+   * @private
+   */
+  setEnabled_(paginationButton, isVisible) {
+    const {element} = paginationButton;
+    element.classList.toggle('i-amphtml-story-button-hidden', !isVisible);
```

Comment:
Should this `toggle` and the `toggleAttribute` in this method be within a `mutateElement()`?